1-JavaScript/Week3/homework/ex3-tellFortune.js

```diff
@@ -33,30 +33,28 @@ body, this code is now written once only in a separated function.
 
 // This function should take an array as its parameter and return
 // a randomly selected element as its return value.
-function selectRandomly(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function selectRandomly(array) {
+  const i = Math.floor(Math.random() * array.length);
```

Comment:
```suggestion
  const i = Math.floor(Math.random() * array.length);
```
it's not really best practice to name a variable 'i' or a single letter in general.. make sure that your names are readable and present what a variable does.. Here 'randomNum' or 'randomNumber' would be better 

and actually you could just return Math.floor(Math.random() * array.length); without a variable

Reply:
"Var i"  was as an index but it's not an index Lol, and I wanted to make it easier to read than writing a long text, I appreciate your advice, going to change it,